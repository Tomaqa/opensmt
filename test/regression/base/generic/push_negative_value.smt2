(set-logic QF_UF)
(declare-fun v19 () Bool)
(push -1)
(check-sat)
