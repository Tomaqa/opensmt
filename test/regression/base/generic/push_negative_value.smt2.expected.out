(error "Incorrect push command, value is negative.")

sat
